cluster_assign_roles.rb

```diff
@@ -26,24 +26,29 @@
 require 'uri'
 require_relative 'lib/cluster_data'
 require_relative 'lib/chef_node'
+require 'cluster_def'
 
 class ClusterAssignRoles
   include BACH::ClusterData
   include BACH::ClusterData::ChefNode
 
+  def initialize
+    @cd = BACH::ClusterDef.new
```

Comment:
Does this need to be a global? Chef-BCPC used to use globals extensively and it really hurt modularity (as one could expect).